csrc/mmdeploy/backend_ops/tensorrt/batched_bev_nms/trt_batched_bev_nms.hpp

```diff
@@ -78,4 +78,4 @@ class TRTBatchedBEVNMSCreator : public TRTPluginCreatorBase {
                                             size_t serialLength) TRT_NOEXCEPT override;
 };
 }  // namespace mmdeploy
-#endif  // TRT_BATCHED_NMS_PLUGIN_CUSTOM_H
```

Comment:
Why do we remove this comment?

Reply:
I found `csrc/mmdeploy/backend_ops/tensorrt/batched_rotated_nms/trt_batched_rotated_nms.hpp` also remove this comment. Just follow it.

Reply:
Do we need to keep it?

Reply:
keep it